src/ivoc/ivocvect.cpp

```diff
@@ -1966,15 +1964,14 @@ static Object** v_indvwhere(void* v) {
 }
 
 static Object** v_fill(void* v) {
-    Vect* x = (Vect*) v;
-    int top = x->size() - 1;
```

Comment:
This triggered warnings about wrap-around when size (which is unsigned) is zero.